puppet/manifests/nubots.pp

```diff
@@ -339,6 +663,10 @@
 set(CMAKE_CXX_FLAGS \"\${CMAKE_CXX_FLAGS} ${compile_params}\" CACHE STRING \"\")
 
 set(PLATFORM \"${arch}\" CACHE STRING \"The platform to build for.\" FORCE)
+
+# Set cmake up to look in our pkgconfig directories
```

Comment:
Should this be in a separate PR?

Reply:
no, its part of all the gazebo dependencies